tests/test_config.py

```diff
@@ -77,7 +77,7 @@ def test_property_access():
     assert os.getegid() == c.gid
 
     # Proc name
-    assert "gunicorn" == c.proc_name
+    assert c.proc_name == "gunicorn" # misplaced-comparison-constant
```

Comment:
While I think the old version looks like a PHP code, I think it's perfectly fine as-is. I'd prefer disabling misplaced-comparison-constant check.